### ISSUE TITLE
Can't install on PHP 8 (RC3)

### ISSUE THREAD

**Issue by ghost** [Codeception/module-soap#4]
```
- codeception/module-soap 1.0.0 requires php >=5.6.0 <8.0 -> your php version (8.0.0RC3) does not satisfy that requirement.
```

**Comment by Naktibalda** [comment]
I only made my way up to module-queue, so this module will be updated after module-redis.

**Comment by ghost** [comment]
> I only made my way up to module-queue, so this module will be updated after module-redis.

OK. Great =) Hope there are not any real issues to deal with

**Comment by Naktibalda** [comment]
There were some small incompatibilities with PHP 8 in codeception/codeception itself.
I haven't discovered any incompatibilities with PHP 8 in modules, the only change required is to update composer.json to allow PHP 8.
A few modules used assertions deprecated in PHPUnit 9, I had to fix those tests to get them passing on PHP 8.

**Comment by Naktibalda** [comment]
I released version 1.0.1 of this module.
